pages/web-literacy/categories.js

```diff
@@ -0,0 +1,8 @@
+var categories = {
+	PS: "Problem-Solving",
```

Comment:
nit: that is a whole lot of whitespace. Can we turn that into just two spaces indentation?